configs/windows/Agent/windowsservercore/WindowsServerCore1803.Dockerfile

```diff
@@ -107,5 +107,5 @@ ENV CONFIG_FILE="C:/BuildAgent/conf/buildAgent.properties" \
     NUGET_XMLDOC_MODE=skip
 
 USER ContainerAdministrator
-RUN ["setx", "/M", "PATH", "(''{0};{1}\bin;C:\Program Files\Git\cmd;C:\Program Files\Mercurial'' -f $env:PATH, $env:JAVA_HOME)"]
+RUN setx /M PATH ('{0};{1}\bin;C:\Program Files\Git\cmd;C:\Program Files\Mercurial' -f $env:PATH, $env:JAVA_HOME)
```

Comment:
The change of escaping here wouldn't affect the process of image's execution within `containerd`, as `RUN` command is executed during container build process, which is expected to be done on the host containing Docker.